src/Date/DateFormatOptions.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DR\Internationalization\Date;
+
+use IntlDateFormatter;
+
+class DateFormatOptions
+{
+    protected int $dateType = IntlDateFormatter::FULL;
+    protected int $timeType = IntlDateFormatter::FULL;
+    protected int $calendar = IntlDateFormatter::GREGORIAN;
+
+    public function __construct(protected string $locale, protected string $timezone)
+    {
+    }
+
+    public function getLocale(): string
+    {
+        return $this->locale;
+    }
+
+    /**
+     * Set the preferred locale for the formatting. Expects an ietf code (nl_NL, nl_BE, en_GB, etc...). Defaults to system configuration.
```

Comment:
The examples are not ietf codes:
http://tools.ietf.org/html/rfc5646, https://en.wikipedia.org/wiki/IETF_language_tag
See examples:
ietf  en-GB

en_GB is POSIX notation if I'm not mistaking: 
https://pubs.opengroup.org/onlinepubs/009695399/basedefs/xbd_chap07.html#tag_07_02

Just in case its handy, you can get full list like this: 
https://www.php.net/manual/en/resourcebundle.locales.php